content/blog/posts/join-our-team.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I feel like this probably needs to be removed from this PR as well since it already exists in the main site..maybe the filename is different?

Reply:
Removed!